inclusion_connect/accounts/forms.py

```diff
@@ -61,6 +63,18 @@ class Meta:
         model = User
         fields = ("last_name", "first_name", "email")
 
+    def clean_email(self):
+        email = self.cleaned_data["email"]
+        if User.objects.filter(email=email).exists():
+            raise ValidationError(
+                format_html(
+                    "Un compte avec cette adresse e-mail existe déjà, "
+                    '<a href="{}">Cliquez-ici</a> pour vous connecter.',
```

Comment:
```suggestion
                    '<a href="{}">se connecter</a> ?',
```


https://www.systeme-de-design.gouv.fr/elements-d-interface/composants/lien

> - le libellé du lien doit être explicite, c’est à dire exprimer clairement ce qui va se passer pour l’utilisateur. 
> - l’intitulé doit permettre de comprendre la destination ou la fonction du lien (ex: ‘faire ma demande de carte grise’, ‘Télécharger le rapport du CNN (format pdf - 345Ko)’).
> - éviter les intitulés de liens non explicites. Les termes trop génériques, comme “Cliquez ici”, “En savoir plus”, “Lire la suite”, etc, ne permettent pas à l’utilisateur d’identifier la destination ou la fonction du lien. Si vraiment aucune alternative n’est possible , il est possible d’ajouter un attribut title ajoutant des éléments de compréhension à l’intitulé non explicite : <a href=”…” title=”Lire la suite : comment faire ma demande de carte grise”>